sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawContainer.java

```diff
@@ -15,7 +16,12 @@ private RawContainer(
             @JsonProperty("image-name") String imageName,
             @JsonProperty("rlimits") WriteOnceLinkedHashMap<String, RawRLimit> rlimits) {
         this.imageName = imageName;
-        this.rlimits = rlimits;
+
+        if (rlimits != null) {
```

Comment:
Yea, I know.  My point is that those fields aren't null.  They're empty collections.  That's what I'm trying to do here as well.